crates/hir-def/src/item_tree.rs

```diff
@@ -467,13 +468,12 @@ macro_rules! mod_items {
         pub enum GenericModItem {
             $(
                 $(
-                    #[cfg_attr(FALSE, $generic_params)]
+                    #[cfg_attr(ignore_fragment, $generic_params)]
```

Comment:
Can we use `#[cfg(any())]` instead?

Reply:
I like using the condition as the description of what this is even doing tbf 😅 (obviously we'd not need this if `$ignore{...}` was stable